scripts/tso-enum.nse

```diff
@@ -280,6 +294,8 @@ action = function(host, port)
     nmap.set_port_version(host, port)
     return result
   else
+    port.version.extrainfo = "Security: " .. secprod
```

Comment:
If this script is run against a non-TN3270 Telnet service, this will obliterate any existing `extrainfo` field, replacing it with "Could not Initiate TN3270" (the first return statement in `tso_test()`). We should probably change that return to make `secprod` false or nil, then change this condition to check for `secprod` being set. Then append instead of replace `extrainfo` (though check if a different TN3270 script already set this info).